GooglePay/src/main/java/com/braintreepayments/api/googlepay/GooglePayActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.braintreepayments.api.googlepay
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.gms.wallet.AutoResolveHelper
+import com.google.android.gms.wallet.PaymentDataRequest
+import com.google.android.gms.wallet.Wallet
+import com.google.android.gms.wallet.Wallet.WalletOptions
+import com.google.android.gms.wallet.WalletConstants
+
+class GooglePayActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        if (savedInstanceState != null && savedInstanceState.getBoolean(EXTRA_RECREATING)) {
+            return
+        }
+
+        val paymentsClient = Wallet.getPaymentsClient(
+            this, WalletOptions.Builder()
+                .setEnvironment(
+                    intent.getIntExtra(
+                        EXTRA_ENVIRONMENT,
+                        WalletConstants.ENVIRONMENT_TEST
+                    )
+                )
+                .build()
+        )
```

Comment:
random q: Did this code auto format this way? The Java variant was about 3 lines. I feel like Kotlin would be about the same since it's less verbose than Java.

Reply:
Yes, it was auto formatted. The number of parameters are the same as Java version, it looks like the line is broken by function calls and parameters to stay within the line length warning.